### PR TITLE
Update che4z-installing.adoc

### PR DESCRIPTION
**Pull request by DStatWriter**
Updated install to include instructions

Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### What does this PR do?
Improves install guide to include instructions and remove redundant link to secondary page

### What issues does this PR fix or reference?
#15018

### CODE REVIEW

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 1]
```diff
@@ -13,4 +13,46 @@ summary:
 
 :context: installing-che4z
 
-To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page].
\ No newline at end of file
+To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page]..Prerequisites
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
To download and install the Eclipse Che4z plugins, follow the instructions at the link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[Eclipse Che4z page].

.Prerequisites
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 2]
```diff
@@ -13,4 +13,46 @@ summary:
...
+. Install Eclipse Che by running the following command. Replace `__<DATA_FOLDER>__` with the location of user data on the host machine:
++
+[subs="+quotes"]
+----
+$ docker run -it --rm -e CHE_MULTIUSER=true -e CHE_HOST=`hostname -f` -e \
+  CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock -v \
+  /var/run/docker.sock:/var/run/docker.sock -v /__<DATA_FOLDER>__:/data eclipse/che:7.0.0-beta-4.0 start
+----
++
+For more information on configuring Eclipse Che with Docker, see https://www.eclipse.org/che/docs/che-6/docker-multi-user.html[Installation on Docker].
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc:41]: Che 6 docs have now been removed, so we shouldn't link to them.

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 3]
```diff
@@ -13,4 +13,46 @@ summary:
...
+
+. Log in to Che. Use your credentials or the default administrator account (username = `admin`, password = `admin`).
+
+. Select *Workspaces -> Add Workspace*.
+
+. In *Select Stack*, select *view all*, then choose the *Broadcom Mainframe* stack.
+
+. Create and open the workspace.
+
+You created a workspace and can now use the Eclipse Che4z extensions.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc:66]:
```suggestion
A workspace has now been created with Eclipse Che4z extensions available.
```

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/che-7/extensions/che4z-installing.adoc (hunk 2 above)
Comment (DStatWriter) [reply src/main/pages/che-7/extensions/che4z-installing.adoc:41]: Is the content covering this point (configuring eclipse che with docker) now covered elsewhere or is it now deprecated?

File: src/main/pages/che-7/extensions/che4z-installing.adoc (hunk 1 above)
Comment (DStatWriter) [reply src/main/pages/che-7/extensions/che4z-installing.adoc]: Signed-off-by: Daniel Statham Daniel.Statham@Broadcom.com

**Comment by DStatWriter** [comment]
Signed-off-by: Daniel Statham Daniel.Statham@Broadcom.com

**Comment by DStatWriter** [comment]
Closed as unable to address issue following commit

File: src/main/pages/che-7/extensions/che4z-installing.adoc (hunk 2 above)
Comment (l0rd) [reply src/main/pages/che-7/extensions/che4z-installing.adoc:41]: We do not support configuring Che with Docker anymore. You can install Che locally using minikube, minishift, microk8s or docker4mac/docker4windows kubernetes.

**Pull request closed without merging**